src/applications/model/four-elements-streaming-server.cc

```diff
@@ -49,6 +49,7 @@ FourElementsStreamingServer::FourElementsStreamingServer ()
 }
 
 FourElementsStreamingServer::FourElementsStreamingServer (Ipv4Address ip, uint16_t port)
```

Comment:
why are you restricting ip to be only ipv4 while set remote is a generic address? same for gaming streaming server and crazy taxi